src/AppBundle/Controller/Admin/UserController.php

```diff
@@ -135,4 +135,56 @@ protected function newUserAction()
 
         return parent::newAction();
     }
+
+    protected function showUserAction()
+    {
+        $this->dispatch(EasyAdminEvents::PRE_SHOW);
+
+        $id = $this->request->query->get('id');
+        $easyadmin = $this->request->attributes->get('easyadmin');
+        $entity = $easyadmin['item'];
+
+        $fields = $this->entity['show']['fields'];
+        $deleteForm = $this->createDeleteForm($this->entity['name'], $id);
+
+        $lectures = [];
+
+        if ($entity->isLector()) {
+            $em = $this->getDoctrine()->getManager();
+
+            $lectures = $em->getRepository('AppBundle:Lecture')
+                ->findBy(
+                    [
+                        'lecturer' => $entity->getId()
+                    ]
+                );
+        }
+
+        $assignments = [];
+
+        if ($entity->isStudent()) {
```

Comment:
Cia elseif galima daryti, nes $entity negali buti ir lektorius ir studentas.
Dar `$em = $this->getDoctrine()->getManager();` Gali iskelti pries if'a, kad nedubliuoti kodo